src/dialects/mssql/formatter.js

```diff
@@ -1,6 +1,6 @@
 
-var inherits        = require('inherits')
-var Formatter       = require('../../formatter')
+import inherits from 'inherits';
+import Formatter from '../../formatter';
 
 import {assign} from 'lodash'
```

Comment:
as before - the whitespace around `assign` seems to be missing here when compared to the new code